site/_components/card.njk

```diff
@@ -2,33 +2,34 @@
   Renders a clickable card.
 
   Arguments:
-    data: List of objects containing the following attributes.
-      - eyebrow (optional): Card eyebrow.
-      - title: Card title.
-      - body: Card body content.
-      - source (optional): Object containing the information source.
-        - logo: The source logo.
-        - name: The source name. i.e. Google Dev Blog.
-      - url: Card URL.
-    modifiers (optional): Object with modifier styles for the card.
-      - custom {String}: Custom modifier styles for the card.
-      - rounded {Boolean}: Rounded border style. Defaults to true.
-    idPrefix (optional): Additional identifier to prevent repeated IDs if multiple cards share the same title.
+    @param {object} data: List of objects containing the following attributes.
```

Comment:
```suggestion
    @param {object} data: Object to populate the card with
```

Two thoughts here:
* Update description to give a hint as to what it's for
* Remove reference to 'list' as it doesn't appear to accept an array